listenbrainz/tests/integration/test_stats_api.py

```diff
@@ -705,3 +712,108 @@ def test_entity_range_stat_not_calculated(self):
         response = self.client.get(url_for('stats_api_v1.get_release',
                                            user_name=self.user['musicbrainz_id']), query_string={'range': 'year'})
         self.assertEqual(response.status_code, 204)
+
+    def test_listening_activity_stat(self):
+        """ Test to make sure valid response is received """
+        response = self.client.get(url_for('stats_api_v1.get_listening_activity', user_name=self.user['musicbrainz_id']))
+        self.assert200(response)
+        data = json.loads(response.data)['payload']
+
+        sent_from = self.listening_activity_payload['from_ts']
+        received_from = data['from_ts']
+        self.assertEqual(sent_from, received_from)
+        sent_to = self.listening_activity_payload['to_ts']
+        received_to = data['to_ts']
+        self.assertEqual(sent_to, received_to)
+        sent_listening_activty = self.listening_activity_payload['listening_activity']
```

Comment:
Spelling of activity here and in other places.

Reply:
Fixed